docs/root/operations/admin.rst

```diff
@@ -175,11 +175,15 @@ modify different aspects of the server:
 
 .. http:get:: /server_info
 
-  Outputs information about the running server. Sample output looks like:
+  Outputs a JSON message containing nformation about the running server. Sample output looks like:
 
 .. code-block:: none
 
-  envoy 267724/RELEASE live 1571 1571 0
+ {
```

Comment:
Can you force it to output all fields even if primitive fields have the default value? IIRC we added an option to the message to JSON conversion function to support this (take a look at the code).